ruff.toml

```
@@ -56,6 +57,15 @@ ignore = [
 "setuptools/__init__.py" = ["E402"]
 "pkg_resources/__init__.py" = ["E402"]
 
+[lint.isort]
+combine-as-imports = true
+split-on-trailing-comma = false
+# Force Ruff/isort to always import setuptools before distutils in tests as long as distutils_hack is supported
+# This also ensures _distutils_hack is imported before distutils
+# https://github.com/pypa/setuptools/issues/4137
+known-third-party = ["setuptools"]
+known-first-party = ["distutils"]
```

Would this make sense?

```suggestion
section-order = ["future", "standard-library", "eager", "third-party", "first-party", "local-folder", "delayed"]
sections.eager = ["_distutils_hack"]
sections.delayed = ["distutils"]
```

It is more verbose, but it does is more compatible with what isort would do by default, special-casing only `distutils` and `_distutils_hack`

---

I didn't realize I could create custom sections. That is so much better yes !